data/card_catalogue/card-manipulation.json

```diff
@@ -0,0 +1,202 @@
+[
```

Comment:
I moved all the cards that directly reference cards into here from unique/upgradeable